word-api/lib/words.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const htmlToText = require('html-to-text');
+const requestPage = require('request');
+const wordCount = require('wordcount');
+
+/**
+ * Get html from url
+ *
+ * @param  {string} url - url of html to scrape.
+ *
+ * @return {string} - text from html.
+ */
+function getText(url) {
+	return new Promise(function (resolve, reject) {
+		let count = 0;
```

Comment:
Never used?

Reply:
Correct. Removed as it was for functionality moved to the count method.